### ISSUE TITLE
Ability to set the context of the expression

### ISSUE THREAD

**Issue by davideicardi** [dynamicexpresso/DynamicExpresso#18]
See
http://stackoverflow.com/questions/23304321/ability-to-set-the-context-of-the-expression/23308125?noredirect=1#comment35753474_23308125

**Comment by OwnageIsMagic** [comment]
Fixed by #233 ?

**Comment by metoule** [comment]
Yes, it's fixed via #233, by setting a variable named `"this"`. Full example:

```c#
var target = new Interpreter();
target.SetVariable("this", new { FirstName="Homer", LastName="Simpson"});
target.Eval("FirstName").Dump();
```
